features/step_definitions/brexit_check_steps.rb

```diff
@@ -1,6 +1,6 @@
 When "I start the checker" do
-  visit_path "/get-ready-brexit-check"
-  click_link "Start now"
+  visit_path "/transition"
+  click_link "Check what's changing"
```

Comment:
Should this not be "Check what’s changing and sign up for email alerts" ?

Reply:
It could be, but I think the text is likely to be iterated by content people, so was hoping the test would match on a bit of the text so we maybe didn't have to tweak it as much.

Reply:
That makes sense - sorry I didn't realise Capybara fuzzy matches by default which is cool -> https://rubydoc.info/github/jnicklas/capybara/#exactness.

Reply:
ikr!